bubble-wrap.yaml

```diff
@@ -2692,7 +2695,6 @@ layers:
         # building footprints, pre-extrusion
         footprints:
             filter:
-                $zoom: { max: 18 }
```

Comment:
Right, I would like to not double-draw anything... that should be avoidable by updating these filters appropriately.